============================= test session starts ==============================
platform linux -- Python 3.10.6, pytest-7.2.0, pluggy-1.0.0
rootdir: /root/projects/qvm
plugins: anyio-3.6.2
collected 55 items / 54 deselected / 1 selected

tst/qvm/integration/bench_test.py 
------------------- Testing --------------

QVM::selection::costs::	random	0.00010967254638671875
compute_unit_list	[{0, 10, 4}, {2}, {3, 6}, {1, 12}, {5}, {11}]
QVM::selection::costs::	naive	0.0001633167266845703
compute_unit_list	[{0, 10, 4}, {2}, {8, 11}, {1, 12}, {3}, {5}]
QVM::selection::costs::	naive_reverse	0.00021004676818847656
compute_unit_list	[{0, 10, 4}, {5}, {3, 6}, {1, 12}, {11}, {2}]
QVM::selection::costs::	brute_force	0.2175767421722412
compute_unit_list	[{0, 10, 4}, {2}, {8, 11}, {1, 12}, {3}, {5}]
random	0.00010967254638671875	0.6719760974952251	0.7679710388183594
small_first	0.0001633167266845703	0.7620803271206809	0.7667487462361654
large_first	0.00021004676818847656	0.6704003664403828	0.7708617846171061
brute_force	0.2175767421722412	0.7619812746057838	0.7665898005167643
.

=============================== warnings summary ===============================
qcs/tst/qvm/integration/bench_test.py: 12 warnings
  /root/anaconda3/envs/qcs-noise/lib/python3.10/site-packages/qiskit/providers/fake_provider/fake_backend.py:462: PendingDeprecationWarning: "warnings" argument will be deprecated as part of the qiskit-aer 0.12.0 and subsequently removed
    noise_model = NoiseModel.from_backend(self, warnings=False)

qcs/tst/qvm/integration/bench_test.py: 18 warnings
  /root/anaconda3/envs/qcs-noise/lib/python3.10/site-packages/qiskit_aer/noise/device/models.py:365: UserWarning: Device model returned an invalid T_2 relaxation time greater than the theoretical maximum value 2 * T_1 (96966.20180926041 > 2 * 40591.82230473617). Truncating to maximum value.
    warn("Device model returned an invalid T_2 relaxation time greater than"

qcs/tst/qvm/integration/bench_test.py::TestBenchDiffBackendQvmFrpV2::test_bench_multi_methods_multi_metrics[/root/projects/QASMBench/small//adder_n10/adder_n10.qasm,/root/projects/QASMBench/small//deutsch_n2/deutsch_n2.qasm,/root/projects/QASMBench/small//hhl_n7/hhl_n7.qasm,/root/projects/QASMBench/small//qaoa_n6/qaoa_n6.qasm,/root/projects/QASMBench/small//quantumwalks_n2/quantumwalks_n2.qasm,/root/projects/QASMBench/small//variational_n4/variational_n4.qasm-FakeBrooklyn-4-random,small_first,large_first,brute_force]
qcs/tst/qvm/integration/bench_test.py::TestBenchDiffBackendQvmFrpV2::test_bench_multi_methods_multi_metrics[/root/projects/QASMBench/small//adder_n10/adder_n10.qasm,/root/projects/QASMBench/small//deutsch_n2/deutsch_n2.qasm,/root/projects/QASMBench/small//hhl_n7/hhl_n7.qasm,/root/projects/QASMBench/small//qaoa_n6/qaoa_n6.qasm,/root/projects/QASMBench/small//quantumwalks_n2/quantumwalks_n2.qasm,/root/projects/QASMBench/small//variational_n4/variational_n4.qasm-FakeBrooklyn-4-random,small_first,large_first,brute_force]
qcs/tst/qvm/integration/bench_test.py::TestBenchDiffBackendQvmFrpV2::test_bench_multi_methods_multi_metrics[/root/projects/QASMBench/small//adder_n10/adder_n10.qasm,/root/projects/QASMBench/small//deutsch_n2/deutsch_n2.qasm,/root/projects/QASMBench/small//hhl_n7/hhl_n7.qasm,/root/projects/QASMBench/small//qaoa_n6/qaoa_n6.qasm,/root/projects/QASMBench/small//quantumwalks_n2/quantumwalks_n2.qasm,/root/projects/QASMBench/small//variational_n4/variational_n4.qasm-FakeBrooklyn-4-random,small_first,large_first,brute_force]
  /root/anaconda3/envs/qcs-noise/lib/python3.10/site-packages/qiskit_aer/noise/device/models.py:365: UserWarning: Device model returned an invalid T_2 relaxation time greater than the theoretical maximum value 2 * T_1 (9.696620180926042e-05 > 2 * 4.059182230473617e-05). Truncating to maximum value.
    warn("Device model returned an invalid T_2 relaxation time greater than"

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
========== 1 passed, 54 deselected, 33 warnings in 145.32s (0:02:25) ===========
